tests/unit/drivers/rank/test_matches2doc_rank_drivers.py

```diff
@@ -62,14 +63,14 @@ def test_chunk2doc_ranker_driver_mock_exec():
     driver.attach(executor=executor, runtime=None)
     driver()
     assert len(doc.matches) == 4
-    assert doc.matches[0].id == '3' * 24
-    assert doc.matches[0].score.value == -1
-    assert doc.matches[1].id == '2' * 16
-    assert doc.matches[1].score.value == -2
+    assert doc.matches[0].id == '2' * 16
+    assert doc.matches[0].score.value == -4
+    assert doc.matches[1].id == '3' * 24
+    assert doc.matches[1].score.value == -4
     assert doc.matches[2].id == '5' * 16
-    assert doc.matches[2].score.value == -3
+    assert doc.matches[2].score.value == -4
     assert doc.matches[3].id == '4' * 8
-    assert doc.matches[3].score.value == -4
+    assert doc.matches[3].score.value == -12
```

Comment:
Why did the value change?

Reply:
Good question! 
This test uses `executor = MockAbsoluteLengthRanker()` to calculate `distance`, and previously we set `match.length = match_score`, and I changed it to `match.length = match_length`. Meanwhile the doc length is also changed `doc.length = 20`.
Which one do you think makes more sense?

Reply:
Let's first merge the code without the value changes. Then we can do a PR changing how the test itself is designed (which I agree has a huge improvement capacity)

Reply:
Sure! That makes sense, I will restore these changes. @JoanFM

Reply:
Just reverted these values change in the latest commit. 😄